apps/full-site-editing/bin/sync-blog-posts-block.sh

```diff
@@ -3,43 +3,49 @@
 # try whether user passed --release
 if [ -n "$npm_config_release" ]
 then
-    MODE=release
-    NAME=$npm_config_release
-    URL=https://github.com/Automattic/newspack-blocks/releases/download/$NAME/newspack-blocks.zip
+	MODE=release
```

Comment:
I changed the file from spaces to tabs here

Reply:
Oh, odd that Prettier didn't do its job?

Reply:
Prettier doesn't format `*.sh` files 🙂

Reply:
<h1>🤦‍♂ </h1>